elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,501 @@
+.light {
+  /* PRIMARY */
+  --_primary-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-background-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_primary-border-color: transparent;
+  --_primary-border-width: var(--rh-border-width-sm, 1px);
+  --_primary-active-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-active-background-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_primary-active-border-width: var(--rh-border-width-sm, 1px);
+  --_primary-focus-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-focus-background-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_primary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_primary-hover-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-hover-background-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_primary-hover-border-width: var(--rh-border-width-sm, 1px);
+
+  /* DANGER */
+  --_danger-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-background-color: var(--rh-color-red-600, #be0000);
+  --_danger-border-color: transparent;
+  --_danger-border-width: var(--rh-border-width-sm, 1px);
+  --_danger-active-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-active-background-color: #a30000; /* WARNING: non-token value */
+  --_danger-active-border-color: transparent;
+  --_danger-focus-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-focus-background-color: #a30000;  /* WARNING: non-token value */
+  --_danger-focus-border-color: transparent;
+  --_danger-focus-border-width: var(--rh-border-width-md, 2px);
+  --_danger-hover-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-hover-background-color: #a30000; /* WARNING: non-token value */
+  --_danger-hover-border-color: transparent;
+
+  /* SECONDARY */
+  --_secondary-color: var(--rh-color-text-primary-on-light, #151515);
```

Comment:
`--_secondary-color` in light theme should be `--rh-color-interactive-blue-darker`. Sorry for the confusion from my other comment!

Reply:
These should be ready to go

Reply:
Please resolve this thread and formally approve to merge, after verifying in the DP